oshmem/mca/spml/spml.h

```diff
@@ -315,24 +315,37 @@ typedef int (*mca_spml_base_module_send_fn_t)(void *buf,
                                               mca_spml_base_put_mode_t mode);
 
 /**
- * The routine transfers the data asynchronously from the source PE to all PEs
- * in the OpenSHMEM job. The routine returns immediately. The source and target
- * buffers are reusable only after the completion of the routine. After the data
- * is transferred to the target buffers, the counter object is updated atomically.
- * The counter object can be read either using atomic operations such as
- * shmem_atomic_fetch or can use point-to-point synchronization routines such as
- * shmem_wait_until and shmem_test.
- *
- *  @param target      The array of remote PE addresses of the objects being written.
- *  @param source      The array of addresses on the local PE holdng the values
- *                     to be written to all other PEs.
+ *  The routine transfers the data asynchronously from the calling PE to all PEs
+ *  in the OpenSHMEM job. The routine returns immediaitely.
```

Comment:
> The routine transfers the data asynchronously from the calling PE to all PEs in the OpenSHMEM job. The routine returns immediaitely.`

`shmemx_alltoall_global_nb is a nonblocking collective routine, where each PE exchanges “size” bytes of data with all other PEs in the OpenSHMEM job. After invocation, the routine returns immediately and it is completed after a subsequent call to shmem_quiet.  `

Reply:
How about above update ? Calling the routine a collective in the front.